### PR TITLE
chore(intergationTests): apply linter for integration tests package

### PR DESCRIPTION
**Pull request by 0xmad**
Blocked by #882
Related to #845

### CODE REVIEW

File: integrationTests/ts/__tests__/integration.test.ts [hunk 1]
```diff
@@ -29,19 +38,19 @@ import {
...
-import { homedir } from "os";
+import { ITestSuite, Subsidy } from "./utils/interfaces";
 import { expectSubsidy, expectTally, genTestUserCommands, isArm } from "./utils/utils";
-import { genPubKey, genRandomSalt } from "maci-crypto";
-import { existsSync, readFileSync, readdir, unlinkSync } from "fs";
-import { expect } from "chai";
+
+chai.use(chaiAsPromised);
+
+const { expect } = chai;
```
Comment (ctrlc03) [inline integrationTests/ts/__tests__/integration.test.ts:46]: can we keep it in one line and do `import chai, { expect } from "chai";` ?

**Review by ctrlc03 — APPROVED** [review]
Only one comment, great stuff thanks

File: integrationTests/ts/__tests__/integration.test.ts (hunk 1 above)
Comment (0xmad) [reply integrationTests/ts/__tests__/integration.test.ts:46]: Unfortunately, no. Before it I need to promisify chai to make it work with typedoc.

Comment (ctrlc03) [reply integrationTests/ts/__tests__/integration.test.ts:46]: Fair enough!

**Pull request merged**